classes/YARPP_Core.php

```diff
@@ -926,13 +926,13 @@ public function display_related($reference_ID = null, $args = array(), $echo = t
         if ($cache_status !== YARPP_NO_RELATED) {
             $orders = explode(' ', $order);
             $wp_query->query(
-                array(
+                apply_filters('yarpp_wp_query', array(
```

Comment:
There are a couple places `WP_Query` is used in YARPP, so it might be better to mention which function is calling it.
```suggestion
                apply_filters('yarpp_display_related_wp_query', array(
```